JASP-Engine/JASP/R/ttestpairedsamples.R

```diff
@@ -239,7 +253,7 @@ TTestPairedSamples <- function(dataset = NULL, options, perform = "run",
 					})
 
 					## if testing raised an error, we extract information from it
-					if (class(r) != "try-error" && is.na(r$statistic) == FALSE) {
+					if (class(r) != "try-error") {
```

Comment:
if (isTryError(r)) {

By the way I don't like objects names referred to as "r". Perhaps call it fit or result